src/ugenem/src/main.cpp

```diff
@@ -44,6 +44,12 @@ QString loadReport(int argc, char *argv[]) {
 }    // namespace
 
 int main(int argc, char *argv[]) {
+    // User lauches the program manually
+    if (argc == 1) {
+        printf("Use \"ugeneui\" to start Unipro UGENE graphical interface or \"ugenecl\" to use the command-line interface.");
+        return 1;
+    }
+
```

Comment:
According the issue description, in case of GUI presented, a message box should be shown.